libraries/ui-library/src/components/six-input/six-input.tsx

```diff
@@ -205,45 +205,52 @@ export class SixInput {
     this.handleClearClick = this.handleClearClick.bind(this);
     this.handlePasswordToggle = this.handlePasswordToggle.bind(this);
     this.handleSlotChange = this.handleSlotChange.bind(this);
-
-    this.host.shadowRoot.addEventListener('slotchange', this.handleSlotChange);
+    this.host.shadowRoot?.addEventListener('slotchange', this.handleSlotChange);
   }
 
   componentWillLoad() {
-    this.defaultValue = this.value;
+    this.defaultValue = this.getValue();
     this.handleSlotChange();
   }
 
   componentDidLoad() {
-    this.eventListeners.add(this.input, 'invalid', (event) => {
+    const nativeInput = this.nativeInput;
+    if (nativeInput === undefined) {
+      return;
+    }
+    this.eventListeners.add(nativeInput, 'invalid', (event) => {
```

Comment:
Is this not a potential problem: If the "nativeInput" is undefined, the eventListener will never be added, because the lifecycle callback is only called once.